package.json

```diff
@@ -25,7 +25,7 @@
     "preview": "vite preview",
     "start:local-docker": "docker container run --rm -it -p 3000:3000 developer-quiz-site",
     "build": "vite build",
-    "test": "vitest",
+    "test": "vitest --run",
```

Comment:
is there a reason why we are disabling watch mode here?
I think watch mode is really helpful especially when you are make a lot of changes and you want vitest to automatically run  relevant tests when updates are made.